crates/sui-json-rpc/src/read_api.rs

```diff
@@ -612,7 +616,8 @@ impl ReadApiServer for ReadApi {
                 )
             })
             .await
-            .map_err(|e| anyhow!(e))??;
+            .map_err(Error::from)?
```

Comment:
the double `.map_err(Error::from)?` is needed to 1) convert the `JoinError` into `Error`, and to convert the error from `state.get_total_transaction_blocks()` into `Error`

Reply:
Reply

Reply:
Added me

Reply:
Aded

Reply:
Do you have anything meaningful to add lol